pkg/ccl/backupccl/backup_destination.go

```diff
@@ -309,7 +309,7 @@ func readLatestFile(
 		return "", err
 	}
 	defer collection.Close()
-	latestFile, err := collection.ReadFile(ctx, latestFileName)
+	latestFile, err := collection.ReadFile(ctx, latestHistoryDirectory+"/"+latestFileName)
```

Comment:
should `latestHistoryDirectory+"/"+latestFileName` just be another const? It's used in a few places.

Reply:
Done